src/odemis/gui/cont/tabs/fastem_main_tab.py

```diff
@@ -0,0 +1,175 @@
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+
+import collections
+import wx
+
+from odemis.acq.stream import EMStream, FastEMSEMStream
+from odemis.gui import main_xrc
+from odemis.gui.comp.viewport import FastEMMainViewport
+from odemis.gui.cont.fastem_user_settings_panel import FastEMUserSettingsPanel
+from odemis.gui.cont.tabs.tab_bar_controller import TabController
+from odemis.gui.cont.tabs.tab import Tab
+from odemis.gui.cont.tabs.fastem_overview_tab import FastEMOverviewTab
+from odemis.gui.cont.tabs.fastem_acquisition_tab import FastEMAcquisitionTab
+import odemis.gui.cont.views as viewcont
+import odemis.gui.model as guimod
+from odemis.gui.util import call_in_wx_main
+from odemis.model import getVAs
+
+
+class FastEMMainTab(Tab):
+    def __init__(self, name, button, panel, main_frame, main_data):
+        """
+        FASTEM main tab which contains the user settings panel, viewport
+        and the overview and acquistion tabs.
+
+        During creation, the following controllers are created:
+
+        FastEMUserSettingsPanel
+          User settings panel which contains the chamber button, e-beam button
+          and scintillation selection panel.
+
+        FastEMOverviewTab
+          It contains the calibration buttons and overview acquisition controller.
+
+        FastEMAcquisitionTab
+          It contains controls for calibrating the system and acquiring regions of
+          acquisition (ROAs). These are organized in projects.
+
+        TabController
+          Handles FastEMOverviewTab and FastEMAcquisitionTab.
+
+        """
+        tab_data = guimod.MicroscopyGUIData(main_data)
+        super(FastEMMainTab, self).__init__(name, button, panel, main_frame, tab_data)
+        self.set_label("MAIN")
+        # Flag to indicate the tab has been fully initialized or not. Some initialisation
+        # need to wait for the tab to be shown on the first time.
+        self._initialized_after_show = False
+
+        # View Controller
+        self.vp = panel.vp_main
+        assert isinstance(self.vp, FastEMMainViewport)
+        vpv = collections.OrderedDict(
+            [
+                (
+                    self.vp,
+                    {
+                        "name": "Acquisition",
+                        "stage": main_data.stage,  # add to show crosshair
+                        "cls": guimod.StreamView,
+                        "stream_classes": EMStream,
+                    },
+                ),
+            ]
+        )
+        self.view_controller = viewcont.ViewPortController(tab_data, panel, vpv)
+
+        # Single-beam SEM stream
+        hwemt_vanames = ("resolution", "scale", "horizontalFoV")
+        emt_vanames = "dwellTime"
```

Comment:
That should be a Tuple too. `in` sort of work here, but as a string search... which is not what you want.
```suggestion
        emt_vanames = ("dwellTime",)
```